src/microbe_stage/PatchMap.cs

```diff
@@ -17,6 +27,12 @@ public class PatchMap
     [JsonProperty]
     public Dictionary<int, Patch> Patches { get; private set; } = new();
 
+    [JsonProperty]
+    public Dictionary<int, PatchRegion> Regions { get; private set; } = new();
+
+    [JsonProperty]
+    public Dictionary<int, PatchRegion> SpecialRegions { get; private set; } = new();
```

Comment:
Please add a comment explaining what's so "special" about this. There's two very similarly named properties next to each other.